en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/faq.md

```diff
@@ -0,0 +1,177 @@
+# Frequently Asked Questions (FAQs)
+
+## About WSO2 Choreo Connect
+
+### What is WSO2 Choreo Connect?
+
+WSO2 Choreo Connect is a cloud-native, open-source, and developer-centric API Gateway. It provides first-class support for Kubernetes while facilitating an array of API management quality of services (QoS), such as message security, rate-limiting, observability, and message mediation.
+
+### What are the key technologies used underneath WSO2 Choreo Connect?
```

Comment:
Is this really a common question that people ask?

Reply:
Not sure whether it's common but as a developer when I use some kind of a new technology / framework I always check for the underlying technologies used there. And I think it's the same with people who do product evaluations and we can highlight the most trending technologies in API and proxy space like envoy proxy.